tower/src/util/pending.rs

```diff
@@ -0,0 +1,141 @@
+use std::fmt;
+use std::{
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+};
+use tower_service::Service;
+
+/// Returns a new `Pending` for the given future.
+///
```

Comment:
may be worth a comment explaining that the `Unpin` bound is required, because the future will be polled in `poll_ready`, which does not have a pinned receiver...

Reply:
Good point. Fixed in eca981b.